specification/resources/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the Resource API.
 
 ``` yaml
 openapi-type: arm
-tag: package-resources-2021-04
+tag: package-changes-2022-03-01-preview
```

Comment:
Why does the default tag need to be changed? I don't think it is needed to be changed for SDK generation. Can't it be left as `package-resources-<stable-api>`? It was introduced in #18475.